main.css

```diff
@@ -0,0 +1,62 @@
+h1 {
+font-family: Arial, Helvetica, sans-serif;
+background-color: #F9F264;
+line-height: 150px;
+padding-left: 20px;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+img {
+  border: 10px solid #F9F264;
+}
+
+section {
+  margin-top: 100px;
+}
+
+h2 {
+  font-family: "Trebuchet MS", Helvetica, sans-serif;
+  text-transform: uppercase;
+  border-bottom: 2px solid;
+  border-bottom-color: black;
+}
+
+li {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#ingredients li {
+  border-bottom: 1px solid;
+  border-bottom-color: grey;
+}
+
+ul.list {
```

Comment:
You mentioned in slack there was some padding and weren't sure how to get rid of it. Easiest way would be to use the padding style in the `<ul>` element!